guides/doc-Release_notes/topics/foreman-3.2.0.adoc

```diff
@@ -0,0 +1,271 @@
+= Foreman 3.2.0
+
+A full list of changes is available via https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1512[Redmine]
+
+== Foreman
+
+* Add Safemode jail definition for ActiveSupport::TimeWithZone - https://projects.theforeman.org/issues/34571[#34571]
+* Helper logging method how_long - https://projects.theforeman.org/issues/34091[#34091]
+
+=== Security
+
+* Settings defined by DSL are not properly encrypted - https://projects.theforeman.org/issues/34573[#34573]
+* The login page exposes version of the foreman - https://projects.theforeman.org/issues/33417[#33417]
+
+=== Inventory
+
+* The Submit button on a host form redirects to an invalid page - https://projects.theforeman.org/issues/34572[#34572]
+
+=== Web Interface
+
+* Add pagelet mountpoints for hosts list table - https://projects.theforeman.org/issues/34543[#34543]
+* default variant prop is missing from pagination component - https://projects.theforeman.org/issues/34410[#34410]
```

Comment:
Please, watch that the first letter is uppercase where it's a normal word - also elsewhere